src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -487,6 +487,7 @@ internal static class WellKnownTypeNames
         public const string SystemXmlXslXslCompiledTransform = "System.Xml.Xsl.XslCompiledTransform";
         public const string SystemXmlXslXslTransform = "System.Xml.Xsl.XslTransform";
         public const string SystemXmlXslXsltSettings = "System.Xml.Xsl.XsltSettings";
+        public const string SystemSecurityPermissionsSecurityPermissionAttribute = "System.Security.Permissions.SecurityPermissionAttribute";
```

Comment:
Keep the list sorted?

Reply:
I didn't even paid attention it was _finally_ sorted!!! 🎉 🎉

Reply:
Yep, thanks @carlossanlop!

Reply:
😃